__tests__/app.test.js

```diff
@@ -108,3 +108,57 @@ describe('"/api/reviews/:reviews_id', () => {
       });
   });
 });
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("200: responds with array of comments related to given review_id sorted by most recent first", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(3);
+        expect(comments).toBeSortedBy("created_at", {
+          descending: false,
+          coerce: true,
+        });
+        comments.forEach((comment) => {
+          expect(comment).toEqual(

```

Comment:
Instead of using toEqual here, swap this out for a more flexible matcher like toMatchObject or objectContaining. This is because the response will change later in the project 👍